whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/RdapObject.java

```diff
@@ -155,4 +173,16 @@ public String getObjectClassName() {
     public void setObjectClassName(final String value) {
         this.objectClassName = value;
     }
+
+    public void setAutnums(final List<Autnum> autnums){
+        this.autnums = autnums;
+    }
+
+    public void setNetworks(List<Ip> ipv4Networks, List<Ip> ipv6Networks) {
```

Comment:
would it make sense to overload this into two methods -- one for v4 and one for v6? otherwise a developer might be tempted to use a null instead of an empty list, which would go badly.